ntp-daemon/src/nts_key_provider.rs

```diff
@@ -19,7 +19,7 @@ pub async fn spawn(config: KeysetConfig) -> watch::Receiver<Arc<KeySet>> {
             .await
             .unwrap_or_else(|e| Err(std::io::Error::new(std::io::ErrorKind::Other, e)))
             .unwrap_or_else(|e| {
-                error!(error = ?e, "Could not load nts server keys, starting with new set");
+                info!(error = ?e, "Could not load nts server keys, starting with new set");
```

Comment:
I think info is too low here, as this is a problem unless this is the very first time you start the server.

Reply:
choice: warn